waspc/src/Wasp/Generator/Common.hs

```diff
@@ -14,27 +14,16 @@ data ProjectRootDir
 -- | Range of node versions that node packages generated by this generator work correctly with.
 nodeVersionRange :: SV.Range
 nodeVersionRange =
-  SV.rangeFromVersionsIntersection
-    [ (SV.BackwardsCompatibleWith, latestLTSVersion),
-      (SV.LessThanOrEqual, latestLTSExactVersionThatWeKnowWorks)
-    ]
+  SV.rangeFromVersion (SV.BackwardsCompatibleWith, latestLTSVersion)
```

Comment:
I wish Haskell had a consistent formatter. This can all fit in a single line, but it won't do it for you, but will allow it if you do it manually.

Reply:
Good catch! Yeah, that would be nice if it did try to tidy that up in some consistent way (but I also should have seen lol). Will update, thanks 👍🏻